### ISSUE TITLE
Mouse doesnt move camera in Outer Worlds (6.6 upstream regression)

### ISSUE THREAD

**Issue by t14462** [Frogging-Family/wine-tkg-git#331]
Yeah, I cant move head. Tested on stock wine-staging.

**Comment by Tk-Glitch** [comment]
This is an upstream issue with wine staging. It'll hopefully get fixed soon.

**Comment by XaeroVincent** [comment]
@Tk-Glitch Looks to be a patch already that fixes it

https://bugs.winehq.org/show_bug.cgi?id=42631#c69

**Comment by Tk-Glitch** [comment]
The patch has just been pushed to staging. Please try again.

**Comment by sangoku116** [comment]
Unable to test, wine applications do not open since latest patch. 

`wine64 /run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local Settings/Application Data/XIVLauncher/XIVLauncher.exe
wineserver: using server-side synchronization.
002c:fixme:winediag:LdrInitializeThunk Wine TkG (staging)  is a testing version containing experimental patches.
002c:fixme:winediag:LdrInitializeThunk Please don't report bugs about it on winehq.org and use https://github.com/Frogging-Family/wine-tkg-git/issues instead.
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126).
WARNING: radv is not a conformant vulkan implementation, testing use only.
wine: failed to open "/run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local": c0000135
`

**Comment by t14462** [comment]
Fixd

**Comment by Tk-Glitch** [comment]
> Unable to test, wine applications do not open since latest patch.
> 
> `wine64 /run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local Settings/Application Data/XIVLauncher/XIVLauncher.exe wineserver: using server-side synchronization. 002c:fixme:winediag:LdrInitializeThunk Wine TkG (staging) is a testing version containing experimental patches. 002c:fixme:winediag:LdrInitializeThunk Please don't report bugs about it on winehq.org and use https://github.com/Frogging-Family/wine-tkg-git/issues instead. 002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126). WARNING: radv is not a conformant vulkan implementation, testing use only. wine: failed to open "/run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local": c0000135 `

If that's with wine (and not proton), you might need to nuke your prefix. Wine 6.6 has broken backward prefixes compat, which I'm only addressing in proton-tkg because it fixes an indirect critical issue with how proton works.

**Comment by Kron4ek** [comment]
> wine64 /run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local Settings/Application Data/XIVLauncher/XIVLauncher.exe
> wine: failed to open "/run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local"

This is because of have spaces in the path to the exe.  Use double or single quotes.

```
wine64 "/run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local Settings/Application Data/XIVLauncher/XIVLauncher.exe"
```

**Comment by sangoku116** [comment]
> > Unable to test, wine applications do not open since latest patch.
> > `wine64 /run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local Settings/Application Data/XIVLauncher/XIVLauncher.exe wineserver: using server-side synchronization. 002c:fixme:winediag:LdrInitializeThunk Wine TkG (staging) is a testing version containing experimental patches. 002c:fixme:winediag:LdrInitializeThunk Please don't report bugs about it on winehq.org and use https://github.com/Frogging-Family/wine-tkg-git/issues instead. 002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (126). WARNING: radv is not a conformant vulkan implementation, testing use only. wine: failed to open "/run/media/Gamez/lutris/final-fantasy-xiv-online/drive_c/users/zangoku/Local": c0000135 `
> 
> If that's with wine (and not proton), you might need to nuke your prefix. Wine 6.6 has broken backward prefixes compat, which I'm only addressing in proton-tkg because it fixes an indirect critical issue with how proton works.

I made an entirely new prefix and I am still unable to launch any application. I get the following:

`wineserver: using server-side synchronization.
002c:fixme:winediag:LdrInitializeThunk Wine TkG (staging)  is a testing version containing experimental patches.
002c:fixme:winediag:LdrInitializeThunk Please don't report bugs about it on winehq.org and use https://github.com/Frogging-Family/wine-tkg-git/issues instead.
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (126).
0058:err:winedevice:ServiceMain Failed to load L"C:\\windows\\system32\\win32k.sys"
0088:err:winedevice:ServiceMain Failed to load L"C:\\windows\\system32\\win32k.sys"
WARNING: radv is not a conformant vulkan implementation, testing use only.
0024:err:seh:segv_handler_early Got unexpected trap 14 during process initialization
`

**Comment by t14462** [comment]
@Tk-Glitch Add this mouse fix to Community-Patches for 6.6r0

**Comment by Tk-Glitch** [comment]
> @Tk-Glitch Add this mouse fix to Community-Patches for 6.6r0

Not sure what you're talking about?
